rotkehlchen/tests/utils/history_base_entry.py

```diff
@@ -42,14 +47,38 @@ def pop_multiple_keys(serialized_event: dict[str, Any], entry_type: HistoryBaseE
             serialized_event.pop(field)
 
 
+def maybe_group_entries(entries: list[HistoryBaseEntry]) -> list[list[HistoryBaseEntry]]:
+    """Group AssetMovements and SwapEvents by their event_identifier so they can be processed
+    as a unit when adding/editing via the api.
+
+    Can't simply group all entry types by the event_identifier since some other entry types also
+    have duplicate event_identifiers (used to test event grouping when querying history events).
+    """
```

Comment:
this needs rewording to actually state why you need them grouped and you don't need evm_events to be grouped by identifier